readme.md

```diff
@@ -13,55 +11,97 @@
+#### Example: Tic Tac Toe
 
 > "Tic Tac Toe is a game where players try to get three squares in a row."
 
 - Game
 - Players
 - Squares
 
+Putting your idea in a nutshell gives you a starting place for what those objects may be.
+
+Spend three minutes working with a partner to come up with at least three types of objects that you might define when creating the following examples.
+
+> A helpful approach might be to take the "nouns" involved in the application and say they are objects.
+
+#### Facebook
+
 > "Facebook is an app where users can post statuses and add friends."
 
-- Users
-- Statuses
-- Friends
+<details>
+  <summary><strong>Some ideas...</strong></summary>
+
+  - Users
+  - Statuses
+  - Friends
+
+</details>
+
+#### Amazon
 
 > "Amazon is a site where people can order products."
 
-- People
-- Orders
-- Products
+<details>
+  <summary><strong>Some ideas...</strong></summary>
+
+  - People
+  - Orders
+  - Products
+
+</details>
 
-I'm basically taking the nouns and saying they're objects.
+#### A Homework Grading App
 
-*Players*, *Users*, and *People* are all "human" words. To keep things simple, I'll almost always use **"Users"** to refer to human objects that interact with my app.
+<details>
+  <summary><strong>Some ideas...</strong></summary>
 
-#### What might the objects be in...
+</details>
 
-- A GA homework-grading app?
-- A GA attendance-taking app?
-- Uber or Lyft?
+#### An Attendance Taking App
 
-## Our first object (10:10 - 10:20)
+<details>
+  <summary><strong>Some ideas...</strong></summary>
+
+</details>
+
+#### Uber
+
+<details>
+  <summary><strong>Some ideas...</strong></summary>
+
+</details>
+
+## Our First Object (10 minutes / 0:20)
 
 An **object** is an **instance** of a **class**. What does this mean? Say that we have a car. Each of us has a mental model of what a car is: it has four wheels, runs on gas, has a steering wheel that allows us to drive it, etc. This is like a **class**. Now, when we see a car in front of us, this is like an **instance**, it's an actual **object** in front of us. Each **object** has its blueprint, and is an **instance** of that blueprint or **class**.
 
-A **class** is a blueprint from which objects are made. In javascript we used prototypes and constructor functions, which operate somewhat similarly to **classes** in Ruby. Each object made from a class is an instance of that class. Each instance of a class is an object.
+A **class** is a blueprint from which objects are made. In javascript we used classes, which operate very similarly to **classes** -- yes, the same name -- in Ruby. Each object made from a class is an instance of that class. Each instance of a class is an object.
 
-Copy and paste this code into your REPL:
+Let's define a `User` class. We'll be using `binding.pry` to test our code.
+
+```bash
+$ touch.app.rb
+$ gem install pry # run this is you haven't installed pry yet
+```
 
 ```rb
+require "pry"
+
 class User
 
   def set_name_to(some_string)
```

Comment:
They were already in the lesson. I tend to use them but happy to not if that's what the students are used to.